### PR TITLE
Add object types and resolvers

### PR DESCRIPTION
**Pull request by andresriveratoro**

### CODE REVIEW

**Comment by andresriveratoro** [comment]
Help lol

**Comment by jthomps838** [comment]
Did slack cut you out? We been trying to figure out to talk to you?

**Comment by andresriveratoro** [comment]
> Did slack cut you out? We been trying to figure out to talk to you?

Yes, I still have no access to Lambda's Slack. I emailed you guys earlier.

File: client/src/components/CreateInvoiceForm/index.js [hunk 1]
```diff
@@ -274,9 +300,9 @@ export default class index extends Component {
...
           <section className="mid-section">
             <div className="mid-section-left">
-              <div>
+              <div className="address-from">
                 <form onSubmit={this.handleFormSubmit}>
-                  <div>From</div>
+                  <div>FROM</div>
                   <TextArea
                     inputType={"text"}
                     //title={"Invoice From"}
```
Comment (47analogy) [inline client/src/components/CreateInvoiceForm/index.js]: Code looks good! Is this something that going to be not used or just commented out until the Invoice Form is ready ?

**Pull request merged**